easybuild/tools/github.py

```diff
@@ -264,7 +264,10 @@ def fetch_latest_commit_sha(repo, account, branch='master', github_user=None, to
             break
 
     if res is None:
-        raise EasyBuildError("No branch with name %s found in repo %s/%s (%s)", branch, account, repo, data)
+        error_msg = "No branch with name %s found in repo %s/%s" % (branch, account, repo)
+        if len(data) >= GITHUB_MAX_PER_PAGE:
```

Comment:
well, there shouldn't be more than `GITHUB_MAX_PER_PAGE` hits, indeed, so I could also check with `==`, but it doesn't matter...